src/modules/iconSelect/components/IconSelect.js

```diff
@@ -33,22 +33,26 @@ export class IconSelect extends MvuElement {
 			icons: [],
 			color: null,
 			value: null,
-			isCollapsed: true
+			isCollapsed: true,
+			portrait: false
```

Comment:
Please extend the test for default values of `isCollapsed` and `portrait`  -> "contains default values in the mode..."